lib/Serialization/ModuleFileSharedCore.cpp

```diff
@@ -679,6 +682,8 @@ void ModuleFileSharedCore::outputDiagnosticInfo(llvm::raw_ostream &os) const {
      << "', built from "
      << (Bits.IsBuiltFromInterface? "swiftinterface": "source")
      << ", " << (resilient? "resilient": "non-resilient");
+  if (Bits.OnlyHasExportableDecls)
+    os << ", built with -experimental-skip-non-exportable-decls or  experimental-skip-non-inlinable-function-bodies";
```

Comment:
nitpick: two spaces and missing - before "experimental-skip..."

Reply:
Similar to Alexis comment below, `experimental-skip-non-inlinable-function-bodies` should not be mentioned here.  If we take this approach, the bit should only represent whether `-experimental-skip-non-exportable-decls` was present. I don't think `-experimental-skip-non-inlinable-function-bodies` should inhibit package resilience optimization.

Reply:
Opts.SkipNonExportableDecls is set if that flag's passed here:   https://github.com/apple/swift/blob/main/lib/Frontend/ArgsToFrontendOptionsConverter.cpp#L302

Reply:
Only when `-experimental-skip-non-inlinable-function-bodies-is-lazy` is also present. That's just a hack to allow project owners to opt-in without driver support.